src/main/java/org/folio/util/TokenCookieParser.java

```diff
@@ -0,0 +1,26 @@
+package org.folio.util;
+
+import org.folio.rest.impl.LoginAPI;
+
+public class TokenCookieParser {
+  public String accessToken;
+  public String refreshToken;
+
+  public TokenCookieParser(String cookieHeader) {
```

Comment:
The spec https://datatracker.ietf.org/doc/html/rfc6265#section-4.2.1 says:
> cookie-header = "Cookie:" OWS cookie-string OWS
> cookie-string = cookie-pair *( ";" SP cookie-pair )

Trim space and tab from cookieHeader, cookieName and cookieValue: `StringUtils.strip(cookieHeader, " \t");`

Do not use String.trim() because it also removes other whitespace.

Reply:
Ok good catch. I had to refer to https://datatracker.ietf.org/doc/html/rfc6265#section-2.2 where OWS is defined and indeed it seems like it could have a tab char in there.